tests/Platforms/AbstractPlatformTestCase.php

```diff
@@ -777,9 +777,12 @@ public function testGetCreateSchemaSQL(): void
 
     public function testAlterTableChangeQuotedColumn(): void
     {
-        $tableDiff                        = new TableDiff('mytable');
-        $tableDiff->fromTable             = new Table('mytable');
-        $tableDiff->changedColumns['foo'] = new ColumnDiff(
+        $table = new Table('mytable');
+        $table->addColumn('select', 'integer');
```

Comment:
Although this change is necessary for the test to pass, it doesn't identify a breaking change. The original test is invalid since it attempts to alter a non-existing column producing an invalid SQL migration.